SITCOM-1397_Extract_dominant_frequencies_from_sound_waves_in_video_associated_with_vibrations_on_Top-End_Assembly_II.ipynb

```diff
@@ -0,0 +1,1504 @@
+{
```

Comment:
<p>The .mp4 video files are not distributed with the notebook (and shouldn't be) I suggest either to indicate clearly where and how to download them or preferably to start directly from the .wav files as those extracted by Kshitija in the notebooks/tel_and_site/subsys_req_ver/tma directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
The notebook is changed to use directly the audio files from the folder.